build/signedbuild.yml

```diff
@@ -154,9 +146,9 @@ jobs:
       uploadAPIScan: false
       uploadBinSkim: false
       uploadFortifySCA: false
+      uploadFxCop: false
```

Comment:
This was `upoadRoslyn: false` before, so TSA was trying to upload FxCop instead of Roslyn